lib/matplotlib/backend_bases.py

```diff
@@ -1059,7 +1059,7 @@ def __init__(self, interval=None, callbacks=None):
             `remove_callback` can be used.
         """
         self.callbacks = [] if callbacks is None else callbacks.copy()
-        self._interval = 1000 if interval is None else interval
+        self.interval = 1000 if interval is None else interval
```

Comment:
Please add a comment, that this intentionally uses the setter. Otherwise this might get reverted by a cleanup (ok, the test would fail, but it's still better to mention this explicitly here.

Reply:
Done.  Also did the same for single_shot, in case anyone wants to override that property too...